content/md/en/docs/tutorials/connect-relay-and-parachains/open-message-passing-channels.md

```diff
@@ -142,53 +145,149 @@ Now that you have the encoded call, you can configure the request to open a chan
-   Be sure that you construct the relative paths for each instruction from the point of view of the receiving system. 
-   For more information about specifying locations, see [Concrete identifiers](https://github.com/paritytech/xcm-format#concrete-identifiers).
-
+   
    In most cases, you also want to include the following instructions:
    
    - [RefundSurplus](https://github.com/paritytech/xcm-format#refundsurplus) to move any overestimate of fees previously paid using the BuyExecution instruction into a second virtual register called the refunded weight register.
   
-   - [DepositAsset](https://github.com/paritytech/xcm-format#depositasset) to subtract assets from the refunded weight register and deposit on-chain equivalent assets under the ownership of the beneficiary. 
+   - [DepositAsset](https://github.com/paritytech/xcm-format#depositasset) to subtract assets from the refunded weight register and deposit on-chain equivalent assets under the ownership of the beneficiary.
+   
+   Note that each instruction requires you to specify additional information to enable the message recipient to execute the intended XCM instructions.
+   Be sure that you construct the information for each instruction from the point of view of the receiving system.
+
+   Let's take a closer look at the instructions.
+
+### WithdrawAsset instruction
+
+To move assets into the virtual holding register:
+
+1. Select [WithdrawAsset](https://github.com/paritytech/xcm-format#withdrawasset) as the first instruction for this message.
+   
+2. Click **Add item** to identify the on-chain assets to withdraw.
+   
+   ![Add instructions or add information about the instruction](substrate-docs/content/media/images/docs/tutorials/parachains/construction-xcm-instruction.png)
+
+3. Select **Concrete** to use the location of the asset to identify the asset to be withdrawn.
+
+     For more information about specifying asset locations, see [Concrete identifiers](https://github.com/paritytech/xcm-format#concrete-identifiers).  
+
+4. Set **parents: 0** and **interior: Here** to withdraw assets from the sovereign account on the relay chain.
+
+5. Select **Fungible** to identify the asset as a fungible asset.
+
+6. Specify the total fungible assets to withdraw.
+  
+   ![WithdrawAsset and settings](/media/images/docs/tutorials/parachains/withdraw-asset-instruction-settings.png)
+
+### BuyExecution instruction
+
+To pay for execution from assets deposited in the holding register:
+
+1. Click **Add item** to select [BuyExecution](https://github.com/paritytech/xcm-format#buyexecution) as the second instruction for this message.
+    
+2. Select **Concrete** to use the location of the asset to identify the asset to be used to pay for executing XCM instructions.
+
+3. Set **parents: 0** and **interior: Here** to use the assets withdrawn from the sovereign account on the relay chain.
```

Comment:
Seems to be the wrong description. You select `Here`  as the `MultiLocation` here to indicate that you're using the local chain's native asset for execution payment, not because you're "using the assets withdrawn from the sovereign account on the relay chain".

Reply:
Thanks for the correction--while I get this "conceptually" deciphering what to use is where things have gotten confusing (for me anyway).